themes/classic/modules/blocktopmenu/blocktopmenu.tpl

```diff
@@ -1,20 +1,36 @@
 {function name="menu" nodes=[] depth=0}
   {strip}
     {if $nodes|count}
-      <ul data-depth="{$depth}" class="top-menu{if $depth === 0} js-top-menu{/if}">
+      <ul aria-labelledby="dLabel" data-depth="{$depth}" class="top-menu">
         {foreach from=$nodes item=node}
-          <li class="{$node.type}{if $node.current} current{/if}">
-            <a href="{$node.url nofilter}" {if $node.open_in_new_window} target="_blank" {/if} data-depth="{$depth}">{$node.label}</a>
-            <div>
-              {menu nodes=$node.children depth=$node.depth}
-            </div>
-          </li>
+          {if $node.children|count}
+            <li class="{$node.type}{if $node.current} current {/if}">
+              <a {if $depth === 0} data-toggle="dropdown" {/if}
+                 class = "{if $depth >= 0}dropdown-item{/if}"
+                 href  = "{$node.url nofilter}"
+                 {if $node.open_in_new_window} target="_blank" {/if}
+                 data-depth="{$depth}"
+              >
+                {$node.label}
+              </a>
+              <div {if $depth === 0} class="dropdown-menu" {/if}>
+                {menu nodes=$node.children depth=$node.depth}
+              </div>
+            </li>
+          {else}
+            <a href  = "{$node.url nofilter}"
+               {if $node.open_in_new_window} target="_blank" {/if}
```

Comment:
Doing too much PrestaShop haha ;) You can still do the reverse and have a javascript that `preventDefault`'s if you don't want the target="_blank" behaviour :)